### ISSUE TITLE
occasional odd Selector behavior when saving a display layout

### ISSUE THREAD

**Issue by larkin** [nasa/openmct#1304]
Testing with current master, can repro every time.
1. navigate to an existing display layout.
2. select "create > display layout".
3. select "save as..."
4. select the root object (as if you wanted to save the layout in the root object).  

Expected outcome: root object is highlighted and indicates it has been selected.
Actual outcome: root object is not highlighted and does not appear to be selected.

I say "does not appear to be selected" because if you click "Ok" the object will be saved into the root as you would expect.

**Comment by larkin** [comment]
This appears to occur with any object, so long as the object you are currently navigated to is the same type of object as the one which you are attempting to create.

**Comment by VWoeltjen** [comment]
By "the root object", does this mean My Items? I've been unable to reproduce this on master (tried current master, 547696d7973cfc715be0571fa213420dc9866cf7 and master from around the time issue was reported, 8cacff37abed33ec18941f71f5c3e36fe04e8e3d )

**Comment by larkin** [comment]
@VWoeltjen thanks for checking in.  I was able to reproduce it, and then I realized that in my development environment, my root folder was a link (leftover from old times).  Upon fixing that, this works as expected.  It does not seem to apply to other objects that are links, so I'm thinking it's an odd side effect with root objects that are links, which are generally considered invalid.

Upon giving my root a location, it now works properly.  Happy to close.
